src/assets/locales/en/cookie-consent-translations.json

```
@@ -1,25 +1,47 @@
 {
   "consentModal": {
     "title": "Your Privacy Choices",
-    "description": "We use cookies to enhance your experience, analyze traffic, and deliver personalized content. You can choose to accept all cookies or manage your preferences for different categories.",
-    "acceptAllBtn": "Accept All Cookies",
+    "description": "We use cookies and similar technologies to operate the website error-free and for the purposes described below. By clicking 'Accept all cookies', you consent to the use of all cookies. If you click 'Reject all', you reject all cookies that require consent. Details on individual Cookies can be found in the 'Manage cookie preferences' modal. You also have the option of selecting Cookie categories individually in that modal. You can change or revoke your cookie settings at any time on <a href=\"/data-privacy\" target=\"_blank\">the data privacy page</a>.",
```

Since it opens an internal page, I would remove the `target="blank"` here

```suggestion
    "description": "We use cookies and similar technologies to operate the website error-free and for the purposes described below. By clicking 'Accept all cookies', you consent to the use of all cookies. If you click 'Reject all', you reject all cookies that require consent. Details on individual Cookies can be found in the 'Manage cookie preferences' modal. You also have the option of selecting Cookie categories individually in that modal. You can change or revoke your cookie settings at any time on <a href=\"/data-privacy\">the data privacy page</a>.",
```